### DISCUSSION TITLE
Cannot delete one of two users?

### DISCUSSION THREAD

**Discussion by ghost** [ghostfolio/ghostfolio#1999] [General]
Wanted to ask around before opening an issue. 
I currently cannot delete one of two users on my install. This would be the first user created. 

And I'm seeing the following in my logs:
<img width="1049" alt="CleanShot 2023-05-22 at 16 37 47@2x" src="https://github.com/ghostfolio/ghostfolio/assets/117439432/ab90daa8-14b7-4e26-80c7-254d02a48e99">

**Comment by dtslvr** [comment]
Hi @enviousjag

I was able to reproduce this error message by trying to delete a user that has already been deleted. What happens if you refresh the browser? Maybe the user has already been deleted.

**Reply by dtslvr** [reply]
Minor improvement: https://github.com/ghostfolio/ghostfolio/pull/2000

**Reply by ghost** [reply]
I just updated to v1.272.0 and I was able to delete the user without an issue.

**Reply by dtslvr** [reply]
Great! 👍🏻  Thanks for the feedback.
